config/default.yml

```
@@ -1347,10 +1347,11 @@ Layout/SpaceInsideParens:
   StyleGuide: '#spaces-braces'
   Enabled: true
   VersionAdded: '0.49'
-  VersionChanged: '0.55'
+  VersionChanged: '1.19'
```

```suggestion
  VersionChanged: '<<next>>'
```

Sorry should have specified this.

---

Out of curiosity, how is this replaced afterwards? Job that runs on pull_requests?

---

There's a `cut_release` task that replaces all the `<<next>>`s with whatever the next version ends up being.